src/SfResources.ar-AE.resx

```diff
@@ -810,6 +810,12 @@
   <data name="Gantt_ConfirmPredecessorDelete" xml:space="preserve">
     <value>هل أنت متأكد من أنك تريد إزالة رابط التبعية؟</value>
   </data>
+  <data name="Gantt_Indent" xml:space="preserve">
+    <value>مسافة بادئة</value>
+  </data>
+  <data name="Gantt_Outdent" xml:space="preserve">
+    <value>عفا عليها الزمن</value>

```

Comment:
Shows wrong meaning in google translate https://translate.google.co.in/?sl=ar&tl=en&text=%D8%B9%D9%81%D8%A7%20%D8%B9%D9%84%D9%8A%D9%87%D8%A7%20%D8%A7%D9%84%D8%B2%D9%85%D9%86&op=translate